src/main/java/net/pms/formats/Format.java

```diff
@@ -352,6 +352,10 @@ public static String getExtension(String mimetype) {
 			ext = ".mp3";
 		} else if (mimetype.equalsIgnoreCase("audio/vnd.dlna.adts")) {
 			ext = ".m4a";
+		} else if (mimetype.equalsIgnoreCase("audio/wav")
+//				|| mimetype.equalsIgnoreCase("audio/x-ms-wav")
+				) {
+			ext = ".wav";
 		} else if (mimetype.equalsIgnoreCase("audio/l16")) {
 			ext = ".l16";
 			
```

Comment:
@atamariya Concerning L16 an other old extension exist ".sw" that is used fy FFmpeg and that FFmpeg developper Carl Eugen recommended, that's said i don't know how it is supported by most renderers ?